### PR TITLE
New  registry entry for 'National Tax Registry' (HN-RTN)

### PR DESCRIPTION
**Pull request by siwhitehouse**
A new list has been proposed with the code HN-RTN

**List title:** National Tax Registry

New list for the Regsitro Tributario Nacional de Honduras

 Preview the platform with this list at [http://org-id.guide/_preview_branch/HN-RTN](http://org-id.guide/_preview_branch/HN-RTN)  (visiting [http://org-id.guide/list/HN-RTN](http://org-id.guide/list/HN-RTN) when the preview is active or check the files changed options above.

Unless objections are raised, this update will be merged after 7 days.

### CODE REVIEW

**Comment by siwhitehouse** [comment]
No objections have been raised. Kadie, can you review this before I merge it, please/

File: lists/hn/hn-rtn.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
...ns and individuals are obliged to register with the National Tax Registry:\n\n1 All individuals or legal entities, national or foreign, domiciled in the country as taxpayers with tax obligations such as: payment of taxes, fees and contributions.\n2 Taxpayers with formal obligations: for filing different tax returns and withholdings.\n3 Taxpayers with tax obligations:\n* Importers, exporters and those who re-apply wholesale purchase purchase management.\n* The companies legally constituted according to the Commercial Code.\n* Foreign companies that carry out operations in Honduras.\n* Irregular or de facto societies.\n* The Undivided Successions (Undivided Heritage).\n* Charitable institutions and Organized Groups for scientific, political, cultural or sporting purposes without profit and that have employees under their charge.\n* Cooperatives of any degree.\n* The Deconcentrated Institutions.\n* The Decentralized Institutions.\n* Others.\n4. Institutions that must register for control purposes:\n* Secretaries of State\n* Autonomous Institutions\n* Semi-autonomous institutions\n* State Corporations\n* City Halls [1]\n[1] https://sde.gob.hn/registro-tributario-nacional-numerico-rtn/"
```
Comment (kd-ods) [inline lists/hn/hn-rtn.json]: Typo? It doesn't quite read right.

Comment (siwhitehouse) [reply lists/hn/hn-rtn.json]: Sorry I've left this so long. What doesn't read quite right?

Comment (kd-ods) [reply lists/hn/hn-rtn.json]: "The National Tax Registry register and hold National Tax Registry Numbers."

**Pull request merged**